backend/aws_helpers/dynamo_db_utils/dynamo_db_utils.py

```diff
@@ -97,6 +99,24 @@ def get_dynamo_items_by_gsi(table_name: str, gsi_value: Union[int, str]) -> list
     return response_items
 
 
+def to_decimal(o):
```

Comment:
I think we should name this function better. This function seems to go thru all items in a dynamodb entry object and sets the type of things that are flota or ints to Decimal type? @dwu359

Reply:
Yeah that was a temporary function that I put there to convert all floats to Decimal. In the future, we could make a Tabular class that handles tabular data and put that as a method there, or any other way that better refactors that function